ci/do_ci.sh

```diff
@@ -382,6 +382,12 @@ elif [[ "$CI_TARGET" == "fix_spelling_pedantic" ]]; then
   echo "fix_spelling_pedantic..."
   ./tools/spelling/check_spelling_pedantic.py fix
   exit 0
+elif [[ "$CI_TARGET" == "headersplit" ]]; then
+  echo "headersplit_test..."
+  bazel_with_collection //tools/envoy_headersplit:headersplit_test LLVM_CONFIG="${LLVM_ROOT}"/bin/llvm-config
```

Comment:
Why do we still need `LLVM_ROOT` if Clang comes in via Python deps?

Reply:
the environment variable has been removed now